packages/backend/server/src/core/doc/reader.ts

```diff
@@ -85,6 +85,29 @@ export abstract class DocReader {
     return content;
   }
 
+  async getFullDocContent(
+    workspaceId: string,
+    docId: string
+  ): Promise<PageDocContent | null> {
+    const cacheKey = this.cacheKey(workspaceId, docId, true);
+    const cachedResult = await this.cache.get<PageDocContent>(cacheKey);
+    if (cachedResult) {
+      return cachedResult;
+    }
+
+    const content = await this.getDocContentWithoutCache(
```

Comment:
What is the maximum size of the content? If it's too large and not suitable for caching, once the redis cache is full, our queue will have issues.

If it's not called frequently, no cache it on parse full doc content?

Reply:
lgtm